pkg/operator/ceph/pool/controller.go

```diff
@@ -356,6 +356,14 @@ func createPool(context *clusterd.Context, clusterInfo *cephclient.ClusterInfo,
 		return errors.Wrapf(err, "failed to create pool %q", p.Name)
 	}
 
+	logger.Infof("initializing rbd pool %q", p.Name)
+	args := []string{"pool", "init", p.Name}
+	output, err := cephclient.NewRBDCommand(context, clusterInfo, args).Run()
+	if err != nil {
+		return errors.Wrapf(err, "failed to initialize replicated pool %q. %s", p.Name, string(output))
+	}
+	logger.Infof("successfully initialized rbd pool %q", p.Name)
```

Comment:
it could be either replicated or EC?
its decided at  https://github.com/rook/rook/blob/87eee8afbca14f5eea4d5b5ab35bf71e77a88130/pkg/daemon/ceph/client/pool.go#L158 ?

Reply:
Ok so then the error message is incorrect.

Reply:
just `logger.Infof("successfully initialized pool %q", p.Name)`will do ?
or keep it as it is?